browser/sources.gni

```diff
@@ -500,6 +500,7 @@ brave_chrome_browser_public_deps = [
   "//brave/browser:browser_process",
   "//brave/components/brave_sync:constants",
   "//brave/components/variations",
+  "//components/guest_view/buildflags",
```

Comment:
really? Where is this used?

Reply:
otherwise there will be a  compile error
```
 In file included from ../../brave/chromium_src/chrome/browser/prefs/browser_prefs.cc:67:
 In file included from ../../../src/chrome/browser/prefs/browser_prefs.cc:25:
 ../../chrome/browser/chrome_content_browser_client.h:31:10: fatal error: 'components/guest_view/buildflags/buildflags.h' file not found
    31 | #include "components/guest_view/buildflags/buildflags.h"
       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 1 error generated.
```
Happens because there is no public path between `//chrome/browser/prefs:impl` and `//components/guest_view/buildflags:buildflags` targets.

`//components/guest_view/buildflags:buildflags` is deps for `//chrome/browser`,
and `//chrome/browser/prefs:impl` is is also deps for `//chrome/browser`.

Somehow this compile error doesn't happen on the vanilla Chromium.

Reply:
It's probably just luck that that it doesn't happen in chromium. That dep should be in `browser_public_dependencies` so we should ideally create an upstream CL to fix it. At the very least we should have a comment here explaining

Reply:
asked at Chromium Slack.

Tried local full rebuild for Debug arm32 and Release arm64 - I could not see the issue.

So reverted at c387e9fb769265ae6fb332bee2972617a0f2698c

If CI will fail, I will bring it back with the comment. If I'll get response on Chromium Slack, I'll do the corrections according to the response.

Reply:
well, it's going to be an intermittent problem, but since the dep is required by a header in chrome_content_browser_client.h this dep clearly belongs in `browser_public_dependencies` even if it's not currently causing a build failure